lib/tasks/users.rake

```diff
@@ -149,4 +149,9 @@ namespace :users do
 
     UserPermissionMigrator.migrate(source: source_application, target: target_application)
   end
+
+  desc "Encrypts user OTP keys, temporary task to migrate away from unencrypted keys"
+  task encrypt_otp_keys: :environment do
+    User.find_each.where.not(otp_secret_key: nil, &:encrypt)
```

Comment:
This seems to have come out in a strange fashion. I assumed you'd just change the code to be: 

```
    two_factor_users = User.where.not(otp_secret_key: nil)

    two_factor_users.find_each(&:encrypt)
```

I didn't even know it'd work in the way you've written it - it does look rather fishy, shall we just do it the more conventional way?